docs/requirements.md

```diff
@@ -1,5 +1,13 @@
 # System requirements
 
+Running Codacy on a Kubernetes cluster requires the following:
+
+* A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#cluster-resource-requirements)
+* The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) for Kubernetes
+* A [PostgreSQL server](#postgresql-server-setup) accessible from the Kubernetes cluster
+
+## Cluster resource requirements
```

Comment:
This section needs to be improved, for now it contains exactly the same information as before.